cron_job_definitions.go

```diff
@@ -85,3 +85,44 @@ func taskSyncTransactions(ctx context.Context, client *Client) error {
 	}
 	return err
 }
+
+func taskCalculateMetrics(ctx context.Context, client *Client) error {
+	m, enabled := client.Metrics()
```

Comment:
Is there any chance that there we will have enabled metrics in this func?
I think we check it here:
`if _, enabled := c.Metrics(); enabled {
		addJob(
			CronJobNameCalculateMetrics,
			15*time.Second,
			taskCalculateMetrics,
		)
	}`

Reply:
You're right. I've just added this to double check this if someone decided to use this function in some other places. But if you find it redundant, I will remove this `if` statement.